schema/Comment.schema.yaml

```diff
@@ -0,0 +1,23 @@
+title: Comment
+'@id': schema:Comment
+extends: CreativeWork
+role: primary
+status: stable
+category: metadata
+description: A comment on an item, e.g on a Article, or SoftwareSourceCode.
```

Comment:
And this could be any item, up to the most generic Thing? Is a comment really only limited to creative works?

Reply:
No, a comment can be on any `Thing`, because the `about` property allows for that:
https://github.com/stencila/schema/blob/master/schema/CreativeWork.schema.yaml#L10-L20. I just used creative works in the example.

Reply:
gotcha!